package.json

```diff
@@ -39,7 +39,7 @@
     "dist"
   ],
   "peerDependencies": {
-    "webpack": "^3.0.0 || ^4.0.0-alpha.0 || ^4.0.0"
+    "webpack": "^5.0.0-beta.5"
```

Comment:
We should support webpack@4 too

Reply:
This PR is about dropping webpack 4 support, and I expect that these changes will be published under major version update ;)

Reply:
@smelukov better keep compatibility with webpack@4 and webpack@5 some times

Reply:
Also `^5.0.0-beta.6` not working with `^5.0.0-beta.5`

Reply:
> Also ^5.0.0-beta.6 not working with ^5.0.0-beta.5

Not sure that understood you correctly...
I've used `^5.0.0-beta.5` for now and it correctly matched for `5.0.0-beta.6` and below
https://jubianchi.github.io/semver-check/#/^5.0.0-beta.5/5.0.0-beta.6

Reply:
Just put here `^5.0.0` and ignore npm warning

Reply:
Anyway i still think we should support webpack@4 and webpack@5

Reply:
Support webpack 4 as well ;)